package.json

```diff
@@ -11,6 +11,7 @@
     "build:cjs": "esbuild index.js --bundle --outfile=bin/dirt-cheap-rocket.cjs --format=cjs --platform=node --minify"
   },
   "dependencies": {
+    "dotenv": "^16.3.1",
```

Comment:
Do we really need dotenv for loading `.env`?

As of Node.js 20, [we already can do this natively](https://nodejs.org/en/learn/command-line/how-to-read-environment-variables-from-nodejs) by using `--env-file` when invoking the `node` command with the script.

With this, there's no need for dotenv and we can just let the user load the variables however they wish.

Reply:
Updated